### DISCUSSION TITLE
Switching off roads option.

### DISCUSSION THREAD

**Discussion by DodomanHD** [BuildTheEarth/terraplusplus#68] [Ideas]
Hi,
The description of the mod says that it is a fork of the “Terra 1 to 1” mod. In the “Terra 1 to 1” mod there was a function to turn off roads, but I can't find it in the Terraplusplus mod. Could you please check this? I would be very grateful if the function was also available in Terreplusplus.
Kind regards
David

**Comment by SmylerMC** [comment]
Hi David,

Generation of features like roads is entirely configurable in Terraplusplus. Unfortunately, the high degree of configurability means achieving something other than the defaults is quite an involved process that requires dealing with configuration files.

The configuration file for feature generation must be created in `.minecraft/terraplusplus`, and be named `osm.json5`. You can find example configuration files [here](https://github.com/BuildTheEarth/terraplusplus/tree/master/src/main/resources/net/buildtheearth/terraplusplus/dataset/osm). `osm.json5` is the default. In your specific case, `osm_no_roads.json5` would probably be a good starting point, but you can edit it to your liking to get the generation you want. Unfortunately, the settings are applied globally, so all your worlds will be affected once you edit the file.

I hope that helps.
